### PR TITLE
Default to syslog agents and remove syslog adapters

### PR DESCRIPTION
**Pull request by chentom88**
### WHAT is this change about?

We are defaulting to syslog agents and removing syslog adapters in order to move away from scalable syslog.

### What customer problem is being addressed? Use customer persona to define the problem e.g. Alana is unable to...

This is a more efficient architecture and will enable the usage of the aggregate drains feature.

### Please provide any contextual information.

https://www.pivotaltracker.com/story/show/171061392

### Has a cf-deployment including this change passed [cf-acceptance-tests](https://github.com/cloudfoundry/cf-acceptance-tests)?

- [x] YES
- [ ] NO

### Does this PR introduce a breaking change? Please take a moment to read through the examples before answering the question.

- [x] YES - please choose the category from below. Feel free to provide additional details.
  - Adds new syslog agent add on
  - Removes syslog adapter instance group and syslog scheduler jobs
  - Adds new syslog binding cache job
- [ ] NO

### How should this change be described in cf-deployment release notes?

Replace the scalable syslog architecture with the shared-nothing syslog architecture. This architecture is more efficient and will enable the usage of the aggregate drains feature.

### Does this PR introduce a new BOSH release into the base cf-deployment.yml manifest or any ops-files?

- [ ] YES - please specify
- [x] NO

### Does this PR make a change to an experimental or GA'd feature/component?

- [x] experimental feature/component
- [ ] GA'd feature/component

### Please provide Acceptance Criteria for this change?

- Install the `cf drain` cli plugin
- Create an app syslog drain to something like papertrail
- Verify that logs for the app make it to your syslog drain

### What is the level of urgency for publishing this change?

- [ ] **Urgent** - unblocks current or future work
- [x] **Slightly Less than Urgent**

### Tag your pair, your PM, and/or team!

PM: @pianohacker

### CODE REVIEW

File: cf-deployment.yml [hunk 1]
```diff
@@ -2359,13 +2374,9 @@ releases:
   version: 2.28.0
   sha1: 683397ede8132bd343f83a4d8c85e943537151a8
 - name: cf-smoke-tests
-  url: https://bosh.io/d/github.com/cloudfoundry/cf-smoke-tests-release?v=40.0.127
+  url: https://bosh.io/d/github.com/cloudfoundry/cf-smoke-tests-release?v=40.0.125
```
Comment (mxplusb) [inline cf-deployment.yml:2377]: Was this a typo?

**Comment by jamespollard8** [comment]
@chentom88 @cloudfoundry/cf-loggregator looks like this could use some attention. There's a review comment and a merge conflict

File: cf-deployment.yml (hunk 1 above)
Comment (Benjamintf1) [reply cf-deployment.yml]: should be fixed.

**Comment by Benjamintf1** [comment]
We squashed our commits and rebased on develop.

**Comment by jamespollard8** [comment]
Looks good to us and we were able to manually validate that this worked. We'll drop validation instructions on the [tracker story](https://www.pivotaltracker.com/story/show/171284806)

**Pull request merged**